Competition_Task/Competition_Task/Tests/CertificationsTests.cs

```diff
@@ -0,0 +1,224 @@
+﻿using Competition_Task.Pages;
+using Competition_Task.TestData;
+using Competition_Task.Utilities;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
+namespace Competition_Task.Tests
+{
+    [TestFixture]
+    
+    public class CertificationsTests:BaseClass
+    {       
+
+            LoginPage LoginPageObj = new LoginPage();
```

Comment:
make use of constructor please

Reply:
or initialise in setup

Reply:
constructor is used in CertificateTests and EducationTests 
![image](https://github.com/Shilpii01/MarsCompetition_Task/assets/145233624/792d0eb5-f427-4a89-8a33-3d03ac930153)